packages/typescript/src/utils.ts

```
@@ -451,20 +482,41 @@ export const formatMessages = ({
 
     messageStrings.push(messageString);
-  }
+      }
 
-  return messageStrings.join("\n");
-};
+      return messageStrings.join("\n");
+    };
 
-const jsonBlockPattern = /```json\n([\s\S]*?)\n```/;
+    export const formatTimestamp = (messageDate: number) => {
```

Biome (2.4.9)

[error] Shouldn't redeclare 'formatTimestamp'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)